### ISSUE TITLE
Finetune a model like training from scratch, why???

### ISSUE THREAD

**Issue by brisker** [szagoruyko/loadcaffe#89]
Hi, after I trained a model from scratch, I found that the epoch seems not enough, so I reload the model and try to finetune it for a bit more epoches. But why the training loss seems like training from scratch?? I just use this code:
model=torch.load(...)
local parameters, gradParameters = model:getParameters()
...
The other codes are just the same as the code for training from scratch. Why this happens? I am really confused about this. Could anybody give some advice?

**Comment by Fourure** [comment]
Pre-trained networks used, in general, some preprocessing for the input data. Some means are substracted to each RGB channels (depending on the dataset used for training). Also caffe used OpenCv who load images in BRG format instead of RGB. Finally, when you load an image in torch pixels values are between 0 and 1 and in Caffe it's between 0 and 255. So trie to see how to preprocess the data to fit the input expected by the pretrained network. By the way, I've made this mistake also and lost lot of time :-/

Ps: sorry for my english.
